### ISSUE TITLE
{{expr}} in custom templates has trailing period

### ISSUE THREAD

**Issue by JoeyBoolean** [yokoe/vscode-postfix-go#8]
### Example
custom = "f := {{expr}}"

foo().custom => f := foo().

The foo() is followed by a period

**Comment by HamzaAnis** [comment]
OK.
Thank you for pointing it out. We will fix it soon.

**Comment by HamzaAnis** [comment]
Thank you for pointing it out.
We will look at it soon.

**Comment by HamzaAnis** [comment]
Can you please recreate this bug and add more details in you issue? Because it working fine without any trailing period.

**Comment by JoeyBoolean** [comment]
Thank you for this extension btw. Its really great!
Here is the json for the custom template
```json
{
    "name": "handleerr",
    "when": [
      "function-call"
    ],
    "description": "Creates an Internal Error",
    "body": "${1:val}, err := {{expr}}\nif err != nil {\n\tsvc.logger.Print(err)\n\treturn ${2:response}, toolbox.InternalError(err)\n}"
  }
```
This is what the go code looks like before the postfix call

```go
func (svc *service) CreateAccessToken(
	ctx context.Context,
	access AccessToken,
) (AccessToken, error) {

	svc.fixedLengthSession(ctx, access.user, validTo)

}
```
This is how I'm calling the postfix snippet

```go
func (svc *service) CreateAccessToken(
	ctx context.Context,
	access AccessToken,
) (AccessToken, error) {

	svc.fixedLengthSession(ctx, access.user, validTo).handleerr

}
```

And this is what it changes to

```go
func (svc *service) CreateAccessToken(
	ctx context.Context,
	access AccessToken,
) (AccessToken, error) {

val, err := svc.fixedLengthSession(ctx, access.user, validTo).
if err != nil {
	svc.logger.Print(err)
	return response, toolbox.InternalError(err)
}

}
```
The period at the end of this line should not be there
```go
val, err := svc.fixedLengthSession(ctx, access.user, validTo).
```

I suspect its line 36 in the [postfixCompletionProvider](https://github.com/yokoe/vscode-postfix-go/blob/master/src/postfixCompletionProvider.ts#L36)
```js
.map(t => t.buildCompletionItem(code, position, line.text.substring(dotIdx, position.character)))
```
The suffix is the substring starting at the dotidx going to the position character. Not sure if maybe it needs to start at dotidx+1. That seems to fix my issue but I'm not sure if that would cause problems elsewhere.

**Comment by yokoe** [comment]
Sorry for being late, I didn't notice the issue.
I will look into it.

**Comment by yokoe** [comment]
This should be fixed in 0.0.12 which was just released. 
Thank you for reporting.
